treeprint_test.go

```diff
@@ -55,8 +55,8 @@ func TestLevel(t *testing.T) {
 	actual := tree.String()
 	expected := `.
 ├── hello
-│   ├── my friend
```

Comment:
It looks like this had a non-breaking space character (`\u00a0`). I didn't tell the reason why the special character was used, so I changed to the normal space character.

If it was intentional to use the `\u00a0`, I will revert these changes.

Reply:
I was following `tree` utility output, but it's actually strange and unpredictable so let's fallback to normal spaces :)

<img width="793" alt="Screenshot 2020-07-15 at 17 11 35" src="https://user-images.githubusercontent.com/477998/87555594-54fa5800-c6be-11ea-81c4-87a4d19a7c02.png">